--- dreditor.user.js.orig	2010-09-11 16:05:13.000000000 -0400
+++ dreditor.user.js	2010-09-11 18:38:32.000000000 -0400
@@ -1,9 +1,10 @@
+// Modified Dreditor Script
+// This script has been modified by Eric J. Duran (ericduran) to work with Chrome.
 // ==UserScript==
 // @name           Dreditor
 // @namespace      http://drupal.org/project/dreditor
 // @description    Awesomeness for drupal.org.
 // @author         Daniel F. Kudwien (sun)
-// @version        0.1
 // @include        http://drupal.org/node/*
 // @include        http://drupal.org/comment/reply/*
 // @include        http://drupal.org/project/*
@@ -11,8 +12,9 @@
 // ==/UserScript==
 
 // Initialize window objects.
-$ = window.$ = window.jQuery = unsafeWindow.jQuery;
-Drupal = window.Drupal = unsafeWindow.Drupal;
+$ = jQuery;
+var Drupal = Drupal;
+
 // Bail out in (the unlikely) case that JS has been disabled.
 if (Drupal === undefined) {
   return false;
@@ -33,7 +35,7 @@ if (Drupal === undefined) {
 jQuery.extend({
   debug: function () {
     // Setup debug storage in global window. We want to look into it.
-    window.debug = unsafeWindow.debug = window.debug || [];
+    window.debug = debug = window.debug || [];
 
     args = jQuery.makeArray(arguments);
     // Determine data source; this is an object for $variable.debug().
@@ -426,7 +428,8 @@ Drupal.dreditor.form.form = function (o,
   // Override the default submit handler.
   self.submit(function (e) {
     // Invoke the submit handler of the clicked button.
-    var op = e.originalEvent.explicitOriginalTarget.value;
+    //TODO: Need to get the right values As it can be Save or Cancel
+    var op = 'Save';
     if (self.submitHandlers[op]) {
       self.submitHandlers[op](this, self);
     }
@@ -1430,7 +1433,7 @@ Drupal.behaviors.dreditorIssuesFormReset
  *   outside of .ready(), since Dreditor's behaviors were suddenly invoked with
  *   drupal.org's behaviors, i.e., this might depend on browser/GM version.
  */
-Drupal.dreditor.conf = GM_getValue('dreditor.conf', '');
+Drupal.dreditor.conf = '';
 Drupal.dreditor.confInit();
 jQuery(document).ready(function () {
   Drupal.attachBehaviors(this);
@@ -1438,96 +1441,5 @@ jQuery(document).ready(function () {
 
 // GM_setValue() is only allowed in the user script context.
 window.addEventListener('unload', function () {
-  GM_setValue('dreditor.conf', $.param(Drupal.dreditor.conf));
-}, true);
-
-// Add custom stylesheet.
-// @todo Can we load CSS files that ship with this user script?
-GM_addStyle(" \
-#dreditor-wrapper { position: fixed; z-index: 1000; width: 100%; top: 0; } \
-#dreditor { position: relative; width: 100%; height: 100%; background-color: #fff; border: 1px solid #ccc; } \
-#dreditor #bar, #dreditor-actions { width: 230px; padding: 0 10px; font: 10px/18px sans-serif, verdana, tahoma, arial; } \
-#dreditor #bar { position: absolute; height: 100%; } \
-#dreditor-actions { background-color: #fff; bottom: 0; padding-top: 5px; padding-bottom: 5px; position: absolute; } \
-.dreditor-button, #content a.dreditor-button { background: transparent url(/sites/all/themes/bluebeach/header-back.png) repeat-x 0 -30px; border: 1px solid #06c; color: #fff; cursor: pointer; font: 11px sans-serif, verdana, tahoma, arial; font-weight: bold; padding: 1px 9px; text-transform: uppercase; text-decoration: none; -moz-border-radius: 9px; -webkit-border-radius: 9px; border-radius: 9px; } \
-.dreditor-button:hover, #content a.dreditor-button:hover { background-position: 0 0; } \
-#dreditor .dreditor-button { margin: 0 0.5em 0 0; } \
-.dreditor-patchreview-processed .dreditor-button { margin-left: 1em; } \
-#dreditor #menu { margin: 0; max-height: 30%; overflow-y: scroll; padding: 0; } \
-#dreditor #menu li { list-style: none; margin: 0; overflow: hidden; padding: 0 0.5em 0 0; white-space: nowrap; } \
-#dreditor #menu li li { padding: 0 0 0 1em; } \
-#dreditor #menu > li > a { display: block; padding: 0 0 0 0.2em; background-color: #f0f0f0; } \
-#dreditor a { text-decoration: none; } \
-#dreditor .form-textarea { width: 100%; height: 12em; font: 13px 'courier new', courier, 'lucida console'; color: #000; } \
-#dreditor-content { margin-left: 250px; border-left: 1px solid #ccc; overflow: scroll; height: 100%; } \
-#dreditor-content, pre { font: 13px 'courier new', courier, 'lucida console'; } \
-#dreditor #code { background: transparent url(/sites/all/themes/bluebeach/shade.png) repeat-y scroll 50.7em 0; padding-left: 10px; } \
-#dreditor #code pre { background-color: transparent; border: 0; margin: 0; padding: 0; } \
-#dreditor #code pre span { display: inline-block; margin-left: 1px; width: 2px; height: 7px; background-color: #ddd; } \
-#dreditor #code pre span.error { background-color: #f99; line-height: 100%; width: auto; height: auto; border: 0; } \
-#dreditor #code pre span.error.eof { color: #fff; background-color: #f66; } \
-#dreditor #code pre span.error.tab { background-color: #fdd; } \
-#dreditor #code pre span.hidden { display: none; } \
-#dreditor #code .file { color: #088; } \
-#dreditor #code .old { color: #c00; } \
-#dreditor #code .new { color: #00c; } \
-#dreditor #code .comment { color: #070; } \
-#dreditor #code .has-comment { background-color: rgba(255, 200, 200, 0.5); } \
-#dreditor #code .selected { background-color: rgba(255, 255, 200, 0.5); } \
-.element-invisible { height: 0; overflow: hidden; position: absolute; } \
-#dreditor-overlay { } \
- \
-.dreditor-actions { overflow: hidden; position: relative; } \
-a.dreditor-application-toggle, #content a.dreditor-application-toggle { display: inline-block; padding: 0.05em 0.3em; line-height: 150%; border: 1px solid #ccc; background-color: #fafcfe; font-weight: normal; text-decoration: none; } \
-#content a.dreditor-application-toggle { float: right; margin: 0 0 0 0.5em; } \
-.dreditor-input { border: 1px solid #ccc; padding: 0.2em 0.3em; font-size: 100%; line-height: 150%; } \
- \
-div.dreditor-issuecount { line-height: 200%; } \
-.dreditor-issuecount a { padding: 0 0.3em; } \
- \
-#content .fieldset-flat { display: block; border: 0; width: auto; padding: 0; } \
-.fieldset-flat > legend { display: none; } \
-#dreditor-issue-data .inline-options .form-item { margin-bottom: 0.3em; } \
-.rounded-corners { padding: 10px; width: auto; -moz-border-radius: 8px; border-radius: 8px; } \
-");
-
-/**
- * Check for new Dreditor versions.
- *
- * GM functions can be invoked from GM environment only.
- */
-dreditorUpdateCheck = function () {
-  if (typeof GM_xmlhttpRequest != 'function') {
-    return;
-  }
-  var version = GM_getValue('version', '');
-  var lastChecked = GM_getValue('update.last', 0);
-  var now = parseInt(new Date() / 1000, 10);
-  // Check every 3 days.
-  var interval = 60 * 60 * 24 * 3;
-  if (lastChecked - now < -interval) {
-    // Whatever happens to this request, remember that we tried.
-    GM_setValue('update.last', now);
-    GM_xmlhttpRequest({
-      method: 'GET',
-      url: 'http://drupalcode.org/viewvc/drupal/contributions/modules/dreditor/CHANGELOG.txt?view=co',
-      onload: function (responseDetails) {
-        if (responseDetails.status == 200) {
-          var newversion = responseDetails.responseText.match(/\$Id.+\$/)[0];
-          if (newversion == version) {
-            return;
-          }
-          var doUpdate = window.confirm('A new version of Dreditor is available. Shall we visit the project page to update?');
-          if (doUpdate) {
-            window.open('http://drupal.org/project/dreditor', 'dreditor');
-            // Let's just assume that we DID update. ;)
-            GM_setValue('version', newversion);
-          }
-        }
-      }
-    });
-  }
-};
-
-dreditorUpdateCheck();
-
+  Drupal.dreditor.conf = $.param(Drupal.dreditor.conf);
+}, true);
\ No newline at end of file
